releasenotes/notes/binder-for-latex-tests-2acdaf7d4a2aee4b.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I dont think there is a need of release note, since this change does not affect the users in any way.

Reply:
Cool, I'll remove the reno.